sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpEventHubClient.cs

```diff
@@ -24,6 +24,11 @@ namespace Azure.Messaging.EventHubs.Amqp
     ///
     internal class AmqpEventHubClient : TransportEventHubClient
     {
+        /// <summary>
+        /// The default scope used for token authentication with EventHubs.
```

Comment:
(extreme) nit:  Please indent the text two spaces.  :)

Reply:
Good catch 👍 :)